### DISCUSSION TITLE
Reveal options to snap a window on hovering over the zoom button on title bar

### DISCUSSION THREAD

**Discussion by Pogimon3** [rxhanson/Rectangle#1336] [Ideas]
I'd love to see a feature hovering over the title bar's zoom button reveals options to snap a window (left half, right half, fullscreen) - inspired by Moom. This would make window snapping faster and more intuitive.

<img width="215" alt="Screenshot 2024-02-22 at 3 37 45 PM" src="https://github.com/rxhanson/Rectangle/assets/140201901/438174d7-74c7-4701-997b-a4049ccdd0f3">

**Comment by rxhanson** [comment]
Thanks for the feedback. I'm guessing that you want more than what is already built into macOS? I think Moom created this feature first, then macOS copied it in. This isn't something I have plans to add into the app myself, but see no reason to turn down a good pull request for it.
